### DISCUSSION TITLE
Missing props on react-navigation

### DISCUSSION THREAD

**Discussion by hirbod** [software-mansion/react-native-screens#1390] [Q&A] [answered]
Hi @kacperkapusciak @WoLewicki,

I hope you're doing good. I would kindly use this discussion here to tinker/ask/remind you, that there are still plenty of new props not available on react-navigation. I am asking myself if the decision to port native-stack over to react-navigation was actually a good one? Because everything related to native-stack happens here, and you guys update this library quite frequently, but changes tend to take plenty of time to land into react-navigation.

Is there anything we could do to optimize this? Should you guys reconsider bringing native-stack back to the mothership?

I see this important props missing:

- useReanimatedTransitionProgress
- transitionDuration
- swipeDirection
- nativeBackButtonDismissalEnabled
- homeIndicatorHidden

Happy to help out!

Best
Hirbod

**Comment by WoLewicki** [comment, marked as answer]
Are you able to spare some time and make a PR in `react-navigation` adding those props there? @kacperkapusciak is not available for next two months and I also do not have much time to take care of it. `nativeBackButtonDismissalEnabled` PR is already there: https://github.com/react-navigation/react-navigation/pull/10354 and other props should be easy to be added. It is also up to @satya164 to find some time to review and eventually merge and release them then.

**Reply by hirbod** [reply]
Ok, I will create a combined PR soon, thanks for your Feedback

**Reply by hirbod** [reply]
https://github.com/react-navigation/react-navigation/pull/10578
